pkg/ccl/streamingccl/streamingest/stream_ingestion_job.go

```diff
@@ -252,8 +253,14 @@ func ingest(ctx context.Context, execCtx sql.JobExecContext, ingestionJob *jobs.
 			return errors.Wrap(err, "failed to update job progress")
 		}
 
-		log.Infof(ctx, "ingestion job %d resumes stream ingestion from start time %s",
-			ingestionJob.ID(), progress.GetStreamIngest().StartTime)
+		if previousHighWater.IsEmpty() {
```

Comment:
`previousHighWater` can be empty if this is the first resumption of the replication job i.e. there is no high water set. We can't always print `initialScanTimestamp` because if we do have a `previousHighWater` (on a subsequent resumption of the replication job) then we want to print that timestamp as the start time. Does that make sense or did i misunderstand you?